ext/intl/dateformat/dateformat.stub.php

```diff
@@ -154,10 +154,11 @@ public static function formatObject($datetime, $format = null, ?string $locale =
 
     /**
      * @param int $offset
+     * @param bool $updateCalendar
```

Comment:
this shouldn't be added since it doesn't have more information than the param type declaration